x-pack/plugins/infra/public/containers/logs/log_summary/log_summary.test.tsx

```diff
@@ -50,22 +56,16 @@ describe('useLogSummary hook', () => {
     await waitForNextUpdate();
 
     expect(fetchLogSummaryMock).toHaveBeenCalledTimes(1);
-    expect(fetchLogSummaryMock).toHaveBeenLastCalledWith(
-      expect.objectContaining({
-        sourceId: 'INITIAL_SOURCE_ID',
-      })
-    );
+    expect(fetchLogSummaryMock.mock.calls[0][0]).toMatchObject({ sourceId: 'INITIAL_SOURCE_ID' });
```

Comment:
Doesn't this change the semantics from "last call" to first call? Can we just use the previous check and add `expect.anything()` in place of the `fetch` argument?

(Applies to all assertions below.)

Reply:
I didn't know about `expect.anything()`! That might be a clean solution.

> Doesn't this change the semantics from "last call" to first call?

No. We reset the mock for each test, so at this point the first call is also the last call.